gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.1-bin.zip
```

Comment:
We should fix this forcing Renovate to always download the `-all` wrapper distribution. I'm fixing this in a separate PR

Reply:
What's the benefit of using `all` over `bin`? I thought using `bin` would be better, if `all` is not necessary

Reply:
Quoting Gradle docs about this:
> use the -all distribution to enable your IDE to enable code-completion and being able to navigate to the Gradle source code.

Reply:
Also, FYI, to force a certain distribution type, it's enough to adding the following line to our root build.gradle:

```
wrapper.distributionType = Wrapper.DistributionType.ALL
```

Reply:
Thanks for the explanation!
As far as I understood, you already making these changes in your branch, or should we change it here?

Reply:
I've not made them yet so for me it's different. If you want to make them here, go ahead. Otherwise we can do that after!

Reply:
Dropped the commit